client/src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,59 @@
+
+import './styles.css';
+import React, { useState } from "react";
+import { Link, useNavigate } from 'react-router-dom';
+import { useLogin } from "../../hooks/useLogin";
+
+export const Login = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
```

Comment:
You can use a hook called `useReducer()` for handling this or even better install a forms management library such as react-hook-forms

Reply:
react-use-forms sounds good here, it'll help you keep this clean